lib/MusicBrainz/Server/Entity/Collection.pm

```diff
@@ -6,7 +6,11 @@ use MusicBrainz::Server::Entity::Types;
 use MusicBrainz::Server::Entity::Util::JSON qw( to_json_array );
 use MusicBrainz::Server::Filters qw( format_wikitext );
 
-extends 'MusicBrainz::Server::Entity::CoreEntity';
+extends 'MusicBrainz::Server::Entity';
+with 'MusicBrainz::Server::Entity::Role::GID';
+with 'MusicBrainz::Server::Entity::Role::Name';
+with 'MusicBrainz::Server::Entity::Role::PendingEdits';
```

Comment:
`editor_collection` doesn't have an `edits_pending` column, so this can be omitted unless I'm missing something

Reply:
Good catch, removed in commit https://github.com/metabrainz/musicbrainz-server/pull/2681/commits/3eea2724ecefc706a24a6f3b9663f2c2dd4fb2c3.